soc_018_migrant_deaths/README.md

```diff
@@ -1,9 +1,13 @@
 ## Migrant Deaths Dataset Near Real-time Script
 This file describes the near real-time script that retrieves and processes the [Missing Migrants - Tracking Deaths Along Migratory Routes dataset](https://missingmigrants.iom.int/) for [display on Resource Watch](https://resourcewatch.org/data/explore/Missing-Migrants).
 
-{Describe how the original data came from the source.}
+This dataset was provided by the source as a CSV file. This CSV was transformed into a table so that it could be uploaded to Carto. In order to transform the data from the CSV to the data table used by Resource Watch, the following changes were made:
```

Comment:
removing the 'transformed' part since the csv format is already a table, unlike a json, which is not necessarily a 2D table format
```suggestion
This dataset was provided by the source as an annual CSV file (one CSV file for each year of data). In order to transform the data from the original CSV file to the data table used by Resource Watch, the following changes were made:
```